DOCKER/Dockerfile

```
@@ -27,28 +27,21 @@ WORKDIR $TMHOME
 # p2p, rpc and prometheus port
 EXPOSE 26656 26657 26660
 
-ENTRYPOINT ["/usr/bin/tendermint"]
-CMD ["node"]
 STOPSIGNAL SIGTERM
 
 ARG BINARY=tendermint
 COPY $BINARY /usr/bin/tendermint
 
-# Create default configuration for docker run.
-RUN /usr/bin/tendermint init && \
-    sed -i \
-      -e 's/^proxy_app\s*=.*/proxy_app = "kvstore"/' \
-      -e 's/^moniker\s*=.*/moniker = "dockernode"/' \
-      -e 's/^addr_book_strict\s*=.*/addr_book_strict = false/' \
-      -e 's/^timeout_commit\s*=.*/timeout_commit = "500ms"/' \
-      -e 's/^index_all_tags\s*=.*/index_all_tags = true/' \
-      -e 's,^laddr = "tcp://127.0.0.1:26657",laddr = "tcp://0.0.0.0:26657",' \
-      -e 's/^prometheus\s*=.*/prometheus = true/' \
-      $TMHOME/config/config.toml && \
-    sed -i \
-      -e 's/^\s*"chain_id":.*/  "chain_id": "dockerchain",/' \
-      $TMHOME/config/genesis.json
+# You can overwrite these before the first run to influence
+# config.json and genesis.json. Additionally, you can override
+# CMD to add parameters to `tendermint node`.
+ENV PROXY_APP=kvstore MONIKER=dockernode CHAIN_ID=dockerchain
+
+COPY ./docker-entrypoint.sh /
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
+CMD ["tendermint", "node"]
```

I'd roll this line back to it's original to keep the feature of starting the blockchain, if no parameters were defined.
```suggestion
CMD ["node"]
```

---

changed in https://github.com/tendermint/tendermint/pull/5087/commits/75bb6c2f68e0d8272b76434031e70ec2aa95d4ff